### DISCUSSION TITLE
Can't click on Control Center Items

### DISCUSSION THREAD

**Discussion by paulodiovani** [FelixKratz/SketchyBar#462] [Q&A]
I can add control center items with  `sketchybar --add alias "Control Center,NAME" right`, but the icons are not clickable, making most of them useless.

I see I can write custom menus and actions, but what I want is to allow the original icon menu and/or action when it is clicked (open wifi settings, show calendar and widgets, etc). Is that possible?

**Comment by Masstronaut** [comment]
Seems to still be the case by default. I have a custom wifi item I've built, but it's not as useful as the native one. I tried using the alias "Control Center,WiFi" but clicking it doesn't open the normal menu making it pretty useless. Is there a config option missing to enable this?

**Comment by lukaslindnermusic** [comment]
any news on this?
